patches/server/0005-Paper-config-files.patch

```diff
@@ -784,6 +784,7 @@ index 0000000000000000000000000000000000000000..73a34b4e378e6012a01c8ac8b0922482
 +        public boolean useDimensionTypeForCustomSpawners = false;
 +        public boolean strictAdvancementDimensionCheck = false;
 +        public IntOr.Default compressionLevel = IntOr.Default.USE_DEFAULT;
++        public List<String> requiredPlugins = List.of();
```

Comment:
Set

Reply:
i just added it as an import cause maybe its gonna be needed in the future